services/migrations/gitea_uploader.go

```diff
@@ -553,7 +553,7 @@ func (g *GiteaLocalUploader) updateGitForPullRequest(pr *base.PullRequest) (head
 			}
 
 			if ok {
-				_, _, err = git.NewCommand(g.ctx, "fetch", remote, pr.Head.Ref).RunStdString(&git.RunOpts{Dir: g.repo.RepoPath()})
+				_, _, err = git.NewCommand(g.ctx, "fetch", "--no-tags", "--", remote, pr.Head.Ref).RunStdString(&git.RunOpts{Dir: g.repo.RepoPath()})
```

Comment:
Are we sure we don't want tags here?

Reply:
I don't think pull request needs tags 🤔

Reply:
By default, any tag that points into the histories being fetched is
       also fetched; the effect is to fetch tags that point at branches that
       you are interested in. This default behavior can be changed by using
       the --tags or --no-tags options or by configuring remote.<name>.tagOpt.
       By using a refspec that fetches tags explicitly, you can fetch tags
       that do not point into branches you are interested in as well.

OKs.